src/reference/config/catalogs/catalog-filesystem.md

```diff
@@ -0,0 +1,65 @@
+---
+shortTitle: filesystem
+description: Zilla runtime filesystem catalog
+category:
+  - Catalog
+---
+
+# filesystem Catalog
+
+::: important Feature is in Incubator
```

Comment:
is this catalog going to be in incubator or enabled in the full release?

Reply:
@jfallows I think this question is yours.